server/src/authorizers/permissions.js

```diff
@@ -47,16 +62,30 @@ const rulesChannelMemberships = {
 const rulesMeasurables = {
   Query: {},
   Mutation: {
-    measurementCreate: and(isAuthenticated, or(isChannelPublic, or(isAdmin, isViewer))),
-    measurableArchive: and(isAuthenticated, measurables.isOwner, not(measurables.isArchived)),
-    measurableUnarchive: and(isAuthenticated, measurables.isOwner, measurables.isArchived),
-    measurableUpdate: and(isAuthenticated, measurables.isOwner),
+    measurementCreate: and(
+      isAuthenticated,
+      or(isChannelPublic, or(isAdmin, isViewer)),
```

Comment:
plz change ``isViewer`` to ``currentUserIsViewer``

Reply:
Updated.